### ISSUE TITLE
Version 0.1.1: Arrow "One step back in results" throws python error

### ISSUE THREAD

**Issue by ninsbl** [geopython/MetaSearch#19]
When browsing search results, klicking on the single arrow back throws a python error:

```
 An error has occured while executing Python code:

Traceback (most recent call last):
  File "C:\Users\ninsbl/.qgis2/python/plugins\MetaSearch\dialogs\maindialog.py", line 572, in navigate
    if res == QMessageBox.Ok:
UnboundLocalError: local variable 'res' referenced before assignment

Python version:
2.7.4 (default, Apr  6 2013, 19:54:46) [MSC v.1500 32 bit (Intel)]

QGIS version:
2.0.1-Dufour Dufour, ebebdf3
```

**Comment by ninsbl** [comment]
Forgot to mention: The plugin continues working nevertheless.

Plattform: Windows 7

**Comment by tomkralidis** [comment]
Thanks for the report.  This is fixed in 476e8c177329055d3f4290a92b3cdc0ce5e6a808 and will be release as part of 0.1.2 (this week).
